section9_dictionary/project/main.py

```diff
@@ -0,0 +1,34 @@
+from art import logo
+import os
+
+print(logo)
+print("Welcome to secret auction program.")
+istherebidders = True
+bidders = {}
+greatest_bid = 0
+
+
+while istherebidders:
+    
+    bidder_name = input("What is your name?: ").lower()                     
+    individual_bid = int(input("What is you bid?: $"))
+    bidders[individual_bid] = bidder_name                                               #Put each bidder and their bid into dictionaries name bidders 
+    isthereanybidders = input("Are there other bidder? Type 'yes' or 'no'.\n").lower()             #Asking user to continous the auction or not. 
+    os.system('cls')
```

Comment:
This is my mistake that I didn't give you this function beforehand, but you did some research and implement it perfectly. outstanding job! You show good potential as a software developer